### ISSUE TITLE
Undefined offset

### ISSUE THREAD

**Issue by marceloxp** [PHLAK/SemVer#13]
```sh
$version = new SemVer\Version()
PHP Notice:  Undefined offset: 4 in /home/.../vendor/phlak/semver/src/Version.php on line 77
PHP Notice:  Undefined offset: 5 in /home/.../vendor/phlak/semver/src/Version.php on line 78
```

**Comment by PHLAK** [comment]
This should be safe to ignore as it's just a notice but this seems easy enough to fix.

https://github.com/PHLAK/SemVer/blob/master/src/Version.php#L82-L83

Should just become

```php
$this->preRelease = $matches[4] ?? null;
$this->build = $matches[5] ?? null;
```

**Comment by PHLAK** [comment]
Resolved by 6a89d1f
